src/minimal-webapi-openidconnect-v2/README.md

```diff
@@ -0,0 +1,121 @@
+# Protected ASP.NET Core minimal API | Microsoft identity platform
+
+The sample code provided here has been created using minimal web API in ASP.NET Core 6.0, and slightly modified to be protected for a single organization using [ASP.NET Core Identity](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/identity?view=aspnetcore-6.0) that interacts with [Microsoft Authentication Library (MSAL)](https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-overview).  In other words, a very minimalist web api is secured by adding an authorization layer before user requests can reach protected resources.  At this point it is expected that the user authentication had already happened, so a token containing user's information is being sent in the request headers to be used in the authorization process.
+
+:link: For more information about how to proctect your projects, please let's take a look at https://docs.microsoft.com/en-us/azure/active-directory/develop/sample-v2-code. To know more about how this sample has been generated, please visit https://docs.microsoft.com/en-us/aspnet/core/tutorials/min-web-api?view=aspnetcore-6.0&tabs=visual-studio-code
+
+## Prerequisites
+
+1. [Download .NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
+
+## Register the web API application in your Azure Active Directory
+
+1. Choose a password for the Azure AD App
+
+   ```bash
+   AZURE_AD_APP_SECRET=<at-least-sixteen-characters-here>
+   ```
+
+1. Register a new Azure AD App
+
+   ```bash
+   AZURE_AD_APP_DETAILS_MINIMAL_API=$(az ad app create --display-name "active-directory-dotnet-minimal-api-aspnetcore" --password ${AZURE_AD_APP_SECRET} --available-to-other-tenants -o json) && \
```

Comment:
this is for single tenants as discussed with the team.

Reply:
done | addressed from [ee782](https://github.com/Azure-Samples/ms-identity-docs-code-csharp/pull/2/commits/ee7824d4f95021335637cff57f1a2f821a5a08bb)

Reply:
Looks good.